README.md

```diff
@@ -379,49 +379,49 @@ This plugin will also import product associations. It's a zero configuration imp
 
 ### Launch data import from CLI
 
-To actually import data you must first create queue items with the **enqueue command** and then you can import them with the **consume command**.
+To actually import data you must first create queue items with the **import command** and then you can import them with the **consume command**.
 
 #### Schedule Akeneo PIM import button
 
 This button allows you to queue a product directly from the admin index page. By adding the product to the queue, it will be updated by the Akeneo PIM at the next consume run.
 
 ![Schedule Akeneo PIM import button](schedule-akeneo-import-button.png)
 
-#### Enqueue command
+#### Import command
 
-To create queue items you can use the `webgriffe:akeneo:enqueue` console command:
+To create queue items you can use the `webgriffe:akeneo:import` console command:
 
 ```bash
-bin/console webgriffe:akeneo:enqueue --since="2020-01-30"
+bin/console webgriffe:akeneo:import --since="2020-01-30"
 ```
 
-This will enqueue all Akeneo entities updated after the provided date.
+This will import all Akeneo entities updated after the provided date.
 
 You can also use a "since file" where to read the since date:
 
 ```bash
 echo "2020-01-30" > var/storage/akeneo-sincefile.txt
-bin/console webgriffe:akeneo:enqueue --since-file="var/storage/akeneo-sincefile.txt"
+bin/console webgriffe:akeneo:import --since-file="var/storage/akeneo-sincefile.txt"
 ```
 
-When run with the since file, the enqueue command will write the current date/time to the since file after the enqueueing process is terminated. This is useful when you put the  enqueue command in cron:
+When run with the since file, the import command will write the current date/time to the since file after the importing process is terminated. This is useful when you put the import command in cron:
 
 ```
-*  * * * * /usr/bin/php /path/to/sylius/bin/console -e prod -q webgriffe:akeneo:enqueue --since-file=/path/to/sylius/var/storage/akeneo-enqueue-sincefile.txt
+*  * * * * /usr/bin/php /path/to/sylius/bin/console -e prod -q webgriffe:akeneo:import --since-file=/path/to/sylius/var/storage/akeneo-import-sincefile.txt
 ```
 
-This way the enqueue command is run repeatedly enqueuing only producs modified since the last command execution.
+This way the import command is run repeatedly importing only products modified since the last command execution.
 
-You can also enqueue items only for specific importers:
+You can also import items only for specific importers:
 
 ```bash
-bin/console webgriffe:akeneno:enqueue --importer="Product" --importer="MyImporter" --since="2020-01-30"
+bin/console webgriffe:akeneno:import --importer="Product" --importer="MyImporter" --since="2020-01-30"
 ```
 
-You can also enqueue items regardless of their last update date:
+You can also import items regardless of their last update date:
 
 ```bash
-bin/console webgriffe:akeneno:enqueue --all
+bin/console webgriffe:akeneno:import --all
 ```
 
 #### Consume command
```

Comment:
This section should be replaced with a statement about Symfony Messenger that must be in place and working to have data actually imported.

Reply:
With Sylius v1.11 Symfony Messenger should be already installed and configured. Anyway, a good portion of the doc should be rewritten.

Reply:
Ok maybe it's enough to remove this "Consume command" paragraph and add an additional sentence to the previous one like:

> Import command assume that Symfony Messenger is working as required since Sylius v1.11.

Reply:
We should also remove the "Browsing queue items in the admin" and "Queue clear" sections.

Reply:
Done!